literal-values.md

```diff
@@ -89,11 +91,63 @@ Numeric literals can also be represented in scientific notation, such as `1.2E3,
 
 For more information, see [Numeric Literals in MySQL](https://dev.mysql.com/doc/refman/5.7/en/number-literals.html).
 
-## NULL values
+## Date and Time Literals
+
+There are several formats for Date and Time literals, such as a string representation or a numeric representation.
+In TiDB, when TiDB expects a Date, it will treat `'2017-08-24'`, `'20170824'`, `20170824` as Date.
+
+The TiDB Date value has the following formats:
+
+* `'YYYY-MM-DD'` or `'YY-MM-DD'`, the `-` separator here is not strict, it can be any punctuation mark.
+For example, `'2017-08-24'`, `'2017&08&24'`, `'2012@12^31'` are all the same.
+The only thing that needs special treatment is the'.' sign, which is used as a decimal point to separate the integer and decimal parts.
```

Comment:
```suggestion
* `'YYYY-MM-DD'` or `'YY-MM-DD'`: The `-` delimiter here is not strict. It can be any punctuation. For example, `'2017-08-24'`, `'2017&08&24'`, `'2012@12^31'` are all valid date formats. The only special punctuation is '.', which is is treated as a decimal point to separate the integer and fractional parts. Date and time can be separated by `T` or a white space. For example, `2017-8-24 10:42:00` and `2017-8-24T10:42:00` represents the same date and time.
```